README.md

```diff
@@ -1,20 +1,102 @@
-# android-boilerplate
+# ultimate-logger-android
+[![](https://jitpack.io/v/ArturBorowy/ultimate-logger-android.svg)](https://jitpack.io/#ArturBorowy/ultimate-logger-android)
 
-Architecture: **MVP**.
+**UltimateLogger** is a library with many useful and customizable logging options.
 
-Uses: RxJava, Dagger, JUnit, Mockito, Robolectric, Moshi, LeakCanary, Retrofit, OkHttp, Glide, Chuck...
+This is **Android** version of UltimateLogger. **JVM version**, which you can use e.g. in desktop Java or Kotlin applications, is available [here](https://github.com/ArturBorowy/ultimate-logger-jvm).
 
-**Still in development.**
+**Feel free to add your suggestions, issues and pull requests!**
 
-Feel free to add your suggestions, issues and pull requests!
+## Quick start guide:
 
-**After downloading the project, find in the code "TODO \`PLACEHOLDER NAME\`", and change placeholders to yours values.**
+0. Add UltimateLogger dependencies to your **build.gradle**:
+```groovy
+dependencies {
+    implementation 'com.github.ArturBorowy:ultimate-logger-android:0.1.0'
+    implementation 'com.github.ArturBorowy:ultimate-logger-mp:0.1.0'
+}
+```
 
-## Goals:
-- 90% code coverage.
+Unless you already have, add **JitPack** repository to your project level **build.gradle**:
 
+```groovy
+allprojects {
+  repositories {
+    ...
+    maven { url 'https://jitpack.io' }
+  }
+}
+```
 
+1. Init library (you probably want to do this in `onCreate()` method of yours `Application` class):
+
+```kotlin
+val shouldLog = true
+val defaultTagSettings = TagSettings(
+        shouldLogFileNameAndLineNum = true,
+        shouldLogClassName = true,
+        shouldLogMethodName = true)
+        
+ALogInitializer.init(shouldLog, defaultTagSettings)
+```
+
+2. Log:
+```kotlin
+ALog.d("Your message to log on debug level.")
+ALog.e(Exception("Your exception to log on error level."))
+```
+**Result:**
+![Basic logging result example](https://i.imgur.com/bWgqjhf.png)
+
+3. Stop library (you probably want to do this in `onDestroy()` method of yours `Application` class)::
+```kotlin
+ALogInitializer.destroy()
+```
+
+## Some advanced features:
+1. **Fully customizable tag**:
+As I have shown before: `ALogInitializer.init(...)` method take TagSettings object as argument:
+
+![TagSettings data structure](https://i.imgur.com/oARBKaw.png)
+
+If you set `shouldLogClassName` of `tagSettings` flag to `true`, you will see class name before message in every line logged. `shouldLogFileNameAndLineNum` have cool feature. If you set this flag `true`, you will see in logcat hyperlink to line of code, where you have used ALog (see in example of **Quick start quide#2**).
```

Comment:
1. Did you mean sth like "shouldLogFileNameAndLineNum is a cool feature too"?
2. If you set this flag to true, you will see a hyperlink to the line of code, where...